packages/tools/devtools/devtools-view/src/components/data-visualization/CommonInterfaces.ts

```diff
@@ -30,3 +31,10 @@ export interface DataVisualizationTreeProps<TNode extends VisualNodeBase = Visua
 	 */
 	node: TNode;
 }
+
+/**
+ * TODO
```

Comment:
Docs :)

Reply:
I think we can remove this since we scappred the multi layer edit for the moment and made SharecCell a value node.

Reply:
But it wasn't removed. With pending build changes things are unhappy - this never specified the return type for edit. Planning to remove in #19683.